slack-bridge/run.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+npm start -- -c /config/config.yaml -p 5858
```

Comment:
I am not too sure, I will double-check this.

Reply:
You are right! Great catch!
I was confused about a similar command needed to generate the slack registration file for homeserver but since we have already committed that generated file, the image's default entry point is sufficient.